src/index.js

```diff
@@ -63,29 +60,8 @@ function search(e) {
 
 // Function to get the name of day
 function getDayFullName(num) {
-  switch (num) {
-    case 0:
-      return "Sunday";
-
-    case 1:
-      return "Monday";
-
-    case 2:
-      return "Tuesday";
-
-    case 3:
-      return "Wednesday";
+  const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  if (num > days.length - 1) num = days.length - 1;
+  return days[num];
```

Comment:
Also, if the objective is simply to get the day name, you can use this built-in alternative

`new Date(exactDate).toLocaleString('en-IN', {weekday:'long'});`